briefcase-forms.rst

```diff
@@ -0,0 +1,169 @@
+**********************************
+Briefcase Usage
+**********************************
+
+:doc:`Install Briefcase <briefcase-install>` before going through this section.
+
+.. _pull-from-aggregate:
+
+Pulling forms from ODK Aggregate
+---------------------------------
+
+- Open *ODK Briefcase*.
+- In the window which opens up select :guilabel:`pull` tab.
+- Select Aggregate 0.9.x or Aggregate 1.0 in the :guilabel:`pull data from` drop-down.
+- Enter the url of your ODK Aggregate server in :guilabel:`url` option. For instance `https://proj1.appspot.com` and click on connect.
+- You can either leave the username blank if the anonymousUser is granted Data Viewer permissions, or specify an ODK Aggregate user (Account Type 'ODK') that has been granted Data Viewer permissions.
+- A list of forms will be displayed. Select the ones you want to pull and click on :guilabel:`pull` button at the bottom of window. The forms will be pulled to ODK Briefcase Storage on your local system. 
+
+.. warning::
+
+ ODK Briefcase will abort a pull attempt if the local copy of the form definition file differs in any way from the copy being pulled from the server or ODK Collect device.
+
+.. tip::
+
+ In the above case, if the changes only affect the question text and do not alter the structure of the collected data (or change the formId or version), you can create a new ODK Briefcase Storage area, pull data into that, then manually copy the instances from there into your original storage area.
+
+-----------
+
+.. _push-to-aggregate:
+
+Pushing forms to ODK Aggregate
+--------------------------------
+
+- Open *ODK Briefcase*.
+- In the window which opens up select :guilabel:`push` tab.
+- Select Aggregate 1.0 in the :guilabel:`push data to` drop-down.
+- Enter the url of your ODK Aggregate server in :guilabel:`url` option. For instance `https://proj1.appspot.com` and click on connect.
+- You can either leave the username blank if the anonymousUser is granted Data Viewer permissions, or specify an ODK Aggregate user (Account Type 'ODK') that has been granted Data Viewer permissions.
+- A list of forms will be displayed. Select the ones you want to push and click on :guilabel:`push` button at the bottom of window. The forms will be pushed to ODK Aggregate Server.
+
+
+.. warning::
+
+  ODK Briefcase will abort a push attempt if the form definition already on the ODK Aggregate server differs in any way from the form definition in the ODK Briefcase Storage area.
+
+.. tip::
+
+  If the form definition has changed but the changes do not affect the data model, you can make a copy of the storage area, replace the copy's form definition file with that from the server, and then push the data up from this storage area copy.
+
+-----------
```

Comment:
I don't understand this separator here. Are the notes that follow it part of the preceding section? If so, remove the break. Otherwise, make their relevance more clear.